schema/shared/src/main/scala/zio/blocks/schema/Patch.scala

```diff
@@ -14,21 +14,54 @@ package zio.blocks.schema
  *
  * patch3(Person("Jane", 25)) // Person("John", 30)
  * }}}
+ *
+ * Patches can also be created by diffing two values:
+ * {{{
+ * val person1 = Person("Jane", 25)
+ * val person2 = Person("John", 30)
+ * val patch = Schema[Person].diff(person1, person2)
+ * patch(person1) // Person("John", 30)
+ * }}}
  */
-final case class Patch[S](ops: Vector[Patch.Pair[S, ?]], source: Schema[S]) {
+final case class Patch[S](
+  ops: Vector[Patch.Pair[S, ?]],
```

Comment:
The `Patch.Pair` is the OLD approach and should NOT be retained together with the new approach.